### ISSUE TITLE
Why redirect_to_login when target URL does not require login

### ISSUE THREAD

**Issue by rafrafek** [labd/django-session-timeout#15]
If target URL does not require login and `session_is_expired` is `true`, user should continue to target URL instead of being redirected to login page!

**Comment by Lapeth** [comment]
Agreed. Alternatively, a method should be run (defined in settings?) that takes a request and lets the developer decide whether to redirect.

**Comment by Lapeth** [comment]
Made a fork in case anyone runs into this: https://github.com/Lapeth/django-session-timeout
